Documentation/ceph-cluster-crd.md

```diff
@@ -208,6 +207,29 @@ Some modules will have special configuration to ensure the module is fully funct
 
 * `pg_autoscaler`: Rook will configure all new pools with PG autoscaling by setting: `osd_pool_default_pg_autoscale_mode = on`
 
+### Network Configuration Settings
+
+Configure the network that will be enabled for the cluster and services. This is optional and if not defined then the default SDN will be used.
+
+* `provider`: Specifies the network provider that will be used to connect the network interface. You can choose between `host`, and `multus`.
+* `selectors`: List the network selector(s) that will be used associated by a key.
+
+The selector keys are hard-coded to `public` and `cluster`.
```

Comment:
```suggestion
The selector keys are required to be `public` and `cluster`.
```

Reply:
I'd prefer "required" instead of "hard-coded" everywhere it is mentioned. "hard-coded" is more of an implementation detail IMO.